system/jlib/jcomp.cpp

```diff
@@ -814,7 +814,7 @@ void CppCompiler::setTargetBitLength(unsigned bitlength)
     case GccCppCompiler:
         switch (bitlength)
         {
-        case 32: option = "-m32"; break;
```

Comment:
Will we ever have a situation where the default is _not_ correct?  (IOW 32 on 64bit architecture)?